kotlinx-coroutines-core/common/src/flow/operators/Context.kt

```diff
@@ -190,13 +194,16 @@ public fun <T> Flow<T>.conflate(): Flow<T> = buffer(CONFLATED)
  *     .flowOn(Dispatchers.Default)
  * ```
  *
+ * Note that an instance of [StateFlow] does not have an execution context by itself,
```

Comment:
The other reason is that that documentation should be exaustive. Otherwise, there is no way to figure out which operators behave in a specialized way for `StateFlow`, but to read documentation for all operations.